clusters/build-clusters/01_cluster/README.md

```diff
@@ -77,3 +77,29 @@ $ oc get pod -n openshift-machine-api -l cluster-autoscaler=default
 NAME                                          READY     STATUS    RESTARTS   AGE
 cluster-autoscaler-default-576944b996-h82zp   1/1       Running   0          7m54s
 ```
+
+## Prow Configuration for build cluster
+
+### Deploy admin assets: Manual
+Deploy `./**/admin_*.yaml`.
+
+### Deploy non-admin assets
+Automated by [branch-ci-openshift-release-master-build01-apply](https://github.com/openshift/release/blob/0ac7c4c6559316a5cf40c40ca7f05a0df150ef8d/ci-operator/jobs/openshift/release/openshift-release-master-postsubmits.yaml#L9) and [Prow's config-updater plugin](https://github.com/openshift/release/blob/0ac7c4c6559316a5cf40c40ca7f05a0df150ef8d/core-services/prow/02_config/_plugins.yaml#L198).
+
+### update BW items: Manual
```

Comment:
If possible we should do this with a script -- getting the SA token and uploading to BitWarden can both be done on the CLI

Reply:
Generating the content of attachments for BW should not be hard.

Kind of hesitate to write back to BW even if it is possible:
So far we use the bw-cli only with readonly mode.
Writing is dangerous because we cannot recover easily if mistakes happened.
No git-reset on bw.

The are about 10 files to upload for now. I think it is fine to upload them manually on the BW UI.

Reply:
Resolving ...
We can discuss the details during the implementation.